### ISSUE TITLE
Lobbying

### ISSUE THREAD

**Issue by parkovski** [ethberlin-hackathon/ETHBerlin-Teambuilding#57]
## Pitch

TL;DR: See last paragraph.

I have sort of a love/hate relationship with this idea. Personally, I just want to work on programming languages and music performance workflow apps, have everything be open source and free, and money is replaced by the amount of times you make someone happy.

Anyways, back to reality. I'm sick of the tech industry and the rest of the world getting fucked by Facebook, Google, etc. I'm not really much of a profanity user normally but I expect I'm gonna write a decent amount here, sorry. I'm sick of congress or whatever your country's legislative body is getting told by "industry experts" that some bullshit policy to increase this quarter's profits are in the people's best interests. I'm sick of patent trolls. I'm sick of net neutrality continuing to be an issue and I hope every MPAA and telecom exec that knowingly deceived policy makers burns in hell. I'm sick of "closed gardens" or whatever they call them. Whoever thinks the App Store is not a monopoly is hilariously misled - do you think I can write an app using UIKit and Swift and get more than maybe 2 users without me _and_ you paying Apple? Non-tech people think the issue is about smart phones; whatever - they don't know how platforms and API contracts work, but it's not - it's absolutely about platforms. I can put months or years of work into something where literally the only way to share it is for both of us to pay Apple for the privilege and I think that's bullshit. I'm not even going to touch on anything outside tech or I'd probably write a book. Ok, enough ranting.

I've been through the stages of grief with all this, and as much as I wish I could be an unarrestable super hero that just punches malicious/deceitful powerful people, there's gotta be a rational way to deal with this. The best I can figure is we need an organized voice that speaks directly for the people to the policy makers, and then reports on if the policy makers actually did what we asked. We have to redemocratize our democracies somehow, right?

So a technical perspective - we have three actors essentially. The people, the lobbyists, and the policy makers. For sake of brevity, let's get nerdy here and say we've got a source, an unreliable transport layer, and an unreliable sink. We've dealt with unreliable systems in CS before so I'll leave it to you to apply network theories to this, but basically we want to encourage reliability and boot out unreliable actors, so we'll need delivery notifications. If the lobbyists (the ones we hire to speak on behalf of the people) prove reliable, they get paid! If not, they get fired. The policy makers are outside the scope of this, so the best we can do is report on if they act according to their constituents wishes, and hope we vote out the bad ones.

Part of this, probably the most important part, is neutrality. If some, um, ill-educated people really want their policy maker to ban brown people, yeah we hope they fail, but the system has to just say this is what you asked, this is how your lobbyist conveyed it, these are the actions your policy maker took. Everyone has to have a reason to trust this if it has any chance against oligarchy and misinformation.

I've probably written enough. For the hackathon's sake, I think it'd be cool to build a mini representative democracy where all the hackers can suggest and vote on a couple mock issues and we can test out some good and bad lobbyists and policy makers.

## Team

I'm pretty much self taught, started at about 12 (15 years ago), made a bunch of pranks, became enamored with compilers, languages, and anything to do with them, wrote one with C++ and LLVM in my free time while taking intro to java in college cause forced prerequisites are a really good system, worked at Microsoft and Facebook, accidentally invested in Ethereum, became disillusioned with big companies and went home to make a phone game, Ethereum started picking up and I realized that I wanted to be actively involved in this thing that I put money into during my reckless phase. Missed a few hackathons, but I've wanted to visit Berlin for a while so this one got me to actually go through the motions and apply.

I learned PHP years ago to prank my brother, Node.js when I wanted to do collaborative study guides in college, C++ cause I liked compilers, C#/Unity and shaders because I wanted to write a phone game. Basically, I may not have used Solidity before but I've always been motivated to learn new things for a good project.

I also have a friend working on a legal TL;DR company and if anything happens with this idea I'd like to see if the two could work together.

## Skills wanted

- The usual, solidity, JS, whatever UX framework is less than 15 minutes old.
- Text summarization/machine learning.
- Dealing with crowdsourcing, conflict resolution, group behavior.
- Legal knowledge. I did Grand Jury in AZ, which means I know that if you've got meth in your sock and a cop asks if they can search you, you should probably say no. A surprising amount don't! Know your rights! (But also don't do meth...) Unfortunately that's about the extent of my legal knowledge.
- Belief in real representation and platform neutrality.

## Communication

I pretty much only use GitHub as far as social networks go. I guess you could email me (parkersnell, either gmail or outlook), but it's late in Berlin and jet lagged me probably won't be awake that long before events start tomorrow, so maybe we should just talk then?

**Comment by FollowJack** [comment]
2devs here. Frontend and solidity. Are you already here or still sleeping? :)
